src/components/InputTypes/Select.tsx

```diff
@@ -0,0 +1,183 @@
+/** @jsx jsx */
```

Comment:
Selects can't be closed by mouse presses.

https://user-images.githubusercontent.com/20439493/104824384-113d0280-5849-11eb-88fc-b077d45ec4ea.mov

Reply:
Resolved in b6ae67ff.